api/openapi/core-command.yaml

```diff
@@ -137,6 +148,47 @@ components:
       required:
       - device
       - command
+    MetricsResponse:
+      allOf:
+      - $ref: '#/components/schemas/BaseResponse'
+      description: ""
+      type: object
+      properties:
+        executor:
+          description: "Indicates how the metrics were collected, either by calling a service directly or else within a container."
+          type: string
+        operation:
+          description: "Indicates the operation being performed, in this case \"Metrics\""
+          type: string
+        result:
+          description: "The detailed metrics result"
+          $ref: '#/components/schemas/MetricsResultValue'
+        service:
+          description: "The name of the service handling the request"
+          type: string
+        success:
+          description: "Indicates whether the operation was successful."
+          type: boolean
+      required:
+      - executor
+      - operation
+      - result
+      - service
+      - success
+    MetricsResultValue:
+      description: "Contains the subfields specific to a metrics result."
+      type: object
+      properties:
+        cpuUsedPercent:
+          description: "The average % of CPU utilization"
+          type: number
+        memoryUsed:
+          description: "The total bytes of memory obtained from the OS."
+          type: integer
+        raw:
+          description: "A marshalled response type containing more granular metrics."
+          type: string
+          format: byte
```

Comment:
This is accurate for the metrics endpoint on the system management service but not for the metrics endpoint on the core services.  Initial version of content looks like this:

```
{
  "memAlloc": 869640,
  "memTotalAlloc": 869640,
  "memSys": 6949240,
  "memMallocs": 7750,
  "memFrees": 531,
  "memLiveObjects": 7219,
  "cpuBusyAvg": 80.32117867226148
}
```

Commented here; applies elsewhere.

Reply:
Addressed. Per Cloud's comment, YAML docs are now in a v2 folder.